src/commands/watch.ts

```diff
@@ -0,0 +1,128 @@
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import * as chokidar from 'chokidar';
+import cli from 'cli-ux';
+
+import { FolderDetails } from '../misc';
+import { checkReport, getServerInfo, packageAndZip, uploadApp } from '../misc/deployHelpers';
+import { INormalLoginInfo, IPersonalAccessTokenLoginInfo } from '../misc/interfaces';
+
+export default class Watch extends Command {
```

Comment:
There are some behaviors on the `watch` command that make it a bit confusing to use:

* The `addfiles` and `remfiles` flags are not very descriptive. It is not clear in which format we have to pass a value to those flags. They might be better suited in a configuration file.
* If developer hasn't previously set up the `serverInfo.json` file, the `Getting Server Info...` step fails without instructions of how to fix it; it should show instructions on how to add information to the coniguration file;
* The `ignore` list used in the Watch options is hard coded; this should be configurable by the developer, with a default value if none is provided;

As the three points above are related to having the configuration file, @lolimay and I think it would be more friendly if all the configs for the CLI where included in a file named `.rcappsconfig`. This file would contain a JSON formatted object where you could configure all points above: the data from `serverInfo.json`, `addfiles` and `remfiles` parameters, `ignore` option (which might be the same thing as `remfiles`?). This is interesting because we could use the same `.rcappsconfig` file to add more options in future. Please note this would mean removing the `serverInfo.json` file.

* When the app is already installed, the `watch` command shows an unnecessary warning before trying to update the app; the cli should only show `Installing app...` as the action and find out whether it needs to install or update the app in the background, without showing the "App already installed" error to the developer. Only unpredictable errors should be shown.
* Bonus: make the CLI ask for server info and write it to the config file. The CLI could ask for this information if it finds out it is not configured in the config file. But this is only a "nice to have";